src/config/container.rs

```diff
@@ -309,6 +309,10 @@ pub struct Configuration {
     /// override recursion logic to always attempt recursion, still respects --depth
     #[serde(default)]
     pub force_recursion: bool,
+
+    /// Auto update app feature
+    #[serde(default)]
```

Comment:
```suggestion
    #[serde(skip)]
```

since the user can't set a config file value to true (the app would always exit before doing anything), we can skip serde serialization